package.json

```diff
@@ -9,6 +9,12 @@
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "admin": "cd packages/tech-admin && yarn start",
+    "provider": "cd packages/tech-provider && yarn start",
+    "user":"cd packages/tech-user && yarn start",
+    "node_modules": "(yarn install);(cd packages/tech-admin && yarn install); (cd packages/tech-provider && yarn install);(cd packages/tech-user && yarn install);(cd packages/tech-components && yarn install)"
```

Comment:
node_modules is not needed. remove this and merge

- [ ] remove if needed

Reply:
updated.